File: _assets/scripts/normalizers/animations_fix_old.lua
return function(data)
    local data_attr = python.as_attrgetter(data)
    if data_attr.__contains__("animations") then
        return
    end
    local output = python.builtins.dict()
    output["defined_in"] = data["defined_in"]
    output["animations"] = data
    python.as_attrgetter(output["animations"]).__delitem__("defined_in")
    return output
end

File: _assets/scripts/normalizers/animations_fix_old.py
from typing import Any, TypedDict

__all__ = ["animations_fix_old"]

input_typed_dict = dict[str,Any]
output_typed_dict = TypedDict("output_typed_dict", animations=Any)

def animations_fix_old(data:input_typed_dict|output_typed_dict) -> output_typed_dict|None:
    if "animations" in data:
        return None
    output:output_typed_dict = {"animations": data}
    return output
